packages/framer-motion/src/animation/waapi/create-accelerated-animation.ts

```diff
@@ -45,14 +46,16 @@ export function createAcceleratedAnimation(
     /**
      * If this animation needs pre-generated keyframes then generate.
      */
-    if (options.type === "spring" || !isWaapiSupportedEasing(options.ease)) {
-        /**
-         * If we need to pre-generate keyframes and repeat is infinite then
-         * early return as this will lock the thread.
-         */
-        if (options.repeat === Infinity) return
```

Comment:
Was this fixed in a different PR?

Reply:
Yes and no. `repeat = 0` would have fixed it in this PR. But we did add a 20 second limit a long time ago that did fix this infinite loop. However the true fix in this case is `repeat = 0` - we want to pre-generate the unrepeated keyframes and only repeat them when we play them via WAAPI.

Reply:
I see! Might worth a comment